users/config.rst

```diff
@@ -150,9 +150,7 @@ device
     the configuration file.
 
 versioning
-    Specifies a versioning configuration.
-
-    .. note:: Needs explanation.
+    See :ref:`versioning`
```

Comment:
I'd leave the first sentence and the note in there, while adding this as a `seealso`. The referenced page describes versioning, but not its structure in the config xml